addons/rose/index.js

```diff
@@ -41,6 +42,23 @@ module.exports = {
     app.options.sassOptions.includePaths.push(styleTree);
   },
 
+  /**
+   * Finds the HDS styles folder and includes it into the running
+   * application's `sassOptions.includePaths`.
+   */
+  includeHDSStyles(app) {
+    const stylePath =
+      '../../node_modules/@hashicorp/design-system-tokens/dist/products/css';
+
+    // Setup default sassOptions on the running application
+    app.options.sassOptions = app.options.sassOptions || {};
+    app.options.sassOptions.includePaths =
+      app.options.sassOptions.includePaths || [];
+
+    // Include the addon styles
+    app.options.sassOptions.includePaths.push(stylePath);
+  },
+
```

Comment:
This file still having `includeFlightIcons` function. Assuming the answer to [this question](https://github.com/hashicorp/boundary-ui/pull/1388/files#r1003360249) is a YES. 

Do we still need this function to include flight icons within the app?

Reply:
We still need this function as this function is for rose icons that use ember inline SVG to display. Discussion re this [here](https://hashicorp.slack.com/archives/C012GTH1C4E/p1666739607931999)